include/nbl/asset/utils/IMeshPacker.h

```diff
@@ -351,16 +370,23 @@ class IMeshPacker : public IMeshPackerBase
                 for (uint16_t i = m_minTriangleCountPerMDIData; nextTriangle != triangleArrayEnd && i < m_maxTriangleCountPerMDIData; i++)
                 {
                     // TODO: save the AABB of the MDI batch before it gets "try extended" (will be needed in the future for culling)
+                    if(aabbs != nullptr)
+                        *aabbs = core::aabbox3df(core::vector3df(min.x, min.y, min.z), core::vector3df(max.x, max.y, max.z));
+
                     extendAABB(nextTriangle);
                     float newBatchArea = halfAreaAABB();
                     if (newBatchArea > kGrowthLimit* batchArea)
                         break;
                     nextTriangle++;
                     batchArea = newBatchArea;
+
+                    if(aabbs != nullptr && (nextTriangle == triangleArrayEnd || (i + 1u) >= m_maxTriangleCountPerMDIData))
+                        *aabbs = core::aabbox3df(core::vector3df(min.x, min.y, min.z), core::vector3df(max.x, max.y, max.z));
```

Comment:
move this to line 386 (and move line 389 there too)
```cpp
if (aabbs)
{
    if (nextTriangle == triangleArrayEnd || i==m_maxTriangleCountPerMDIData)
         *aabbs = core::aabbox3df(core::vector3df(min.x, min.y, min.z), core::vector3df(max.x, max.y, max.z));
    aabbs++;
}
```

otherwise you have a bug and no AABB gets written if `m_minTriangleCountPerMDIData==m_maxTriangleCountPerMDIData` or if the batch has less triangles than `m_minTriangleCountPerMDIData`